src/MicrosoftTeams.ts

```diff
@@ -2329,3 +2329,90 @@ export function getChatMembers(
   const messageId = sendMessageRequest(parentWindow, "getChatMembers");
   callbacks[messageId] = callback;
 }
+
+/**
+ * Namespace to interact with the conversational subEntities inside the tab
+ */
+export namespace conversations {
+
+  /**
+   * @private
+   * Hide from docs.
+   * ------
+   */
+  export interface ConversationalSubEntity {
+
+    /**
+     * A function that is called once the conversation Id has been created.
+     */
+    onStartConversation?: (conversationId: string) => void;
+
+    /**
+     * A function that is called if the pane is closed
+     */
+    onCloseConversation?: () => void;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   * --------------
+   * Allows the user to start a conversation with each subentity inside a tab
+   * @param subEntityId The Id of the subEntity where the conversation is taking place
+   * @param title The title of the conversation
+   * @param conversationalSubEntity Callback containing the conversation Id and if the tab pane was closed
+   */
+  export function startConversation(
```

Comment:
Can we merge these two function into on lets say startConversation(conversationRequest: ConversationRequest)  where conversational Id will be optional 
1. if provided we will open pane
2. If null we will create one and open pane return conversation Id.


  export interface ConversationRequest {
   subEntityId: string;
   title: string;
  conversationId?: string
    successCallback?: (conversationId: string) => void;

    failureCallback?: (reason: string) => void;
  }

Reply:
We did discuss about that in the dev design review. But we decided to avoid method overloading to reduce confusion of the developer. And in the final version of the feature we might need to have separate functions.

Reply:
Like the idea of adding a reason in the onCloseConversation callback. Added it

Reply:
Lets use this kind of request object.
export interface ConversationRequest {
   subEntityId: string;
   title: string;
  conversationId?: string
    successCallback/onStartConversation?: (conversationId: string) => void;
    onCloseConversation?: (reason: string) => void;
    failureCallback?: (reason: string) => void;
  }

Reply:
we can discuss about happening one or two separate function